packages/react-components/src/components/search-bar/index.tsx

```diff
@@ -24,8 +24,10 @@ type Props = {
     // leaves the term in context unaffected
     clear?: boolean
     autoFocus?: boolean
+    tag?: ComponentType
```

Comment:
@kopax-polyconseil the build is breaking, I don't think this is the right typing. We'd want to do something more along the lines of. 

```tsx
inputComponent: ElementType<InputComponentProps>
```

This way if you're defining a custom input component, the props are typed. 

I can support this feature. I'll add it to our board